spec/features/recipes/create_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# rubocop:disable Layout/LineLength
+RSpec.describe 'Create recipe' do
+  subject { visit new_recipe_path }
+
+  let!(:user) { create(:user) }
+
+  before do
+    sign_in user
+    subject
+  end
+
+  describe 'render new recipe form' do
+    it 'shows ingredients field' do
+      within('form') do
+        expect(page).to have_field('recipe[ingredients]')
+      end
+    end
+  end
+
+  describe 'create recipe' do
+    context 'with valid data' do
+      let(:ingredients) { 'Harina, Azucar, Pan, Aceite' }
+
+      before do
+        stub_request(:post, 'https://api.openai.com/v1/chat/completions')
+          .with(
+            body: {
+              model: 'gpt-3.5-turbo-1106',
+              messages: [
+                { role: 'system',
+                  content: "Create a recipe using only the ingredients provided. Do not include any other ingredients.\nReturn the response in a JSON, with this structure:\n{\n  \"name\": \"Dish Name\",\n  \"content\": \"Recipe instructions\"\n}\nBe sure the JSON is well formatted without extra spaces or new lines, and do not include any delimiters like ```json. \n" },
```

Comment:
Not a blocker, but to avoid bigger lines you can split it with `\` like:

`content: "Create a recipe using only the ingredients provided. " \`
`"Do not include any other ingredients.\nReturn" \`